pom.xml

```diff
@@ -1,45 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.eclipse.jetty.toolchain</groupId>
     <artifactId>jetty-toolchain</artifactId>
-    <version>1.7</version>
+    <version>1.8-parent-SNAPSHOT</version>
```

Comment:
The child repos should never depend on a SNAPSHOT parent.
Depending on a SNAPSHOT complicates the release process.

This will also allow dependabot to update the parent versions.
AND we'll see when a plugin version update breaks the build of the child repo (which has happened countless times before).
But now we will have to manually fix the incompatible plugin version.

Reply:
@joakime It's a draft/working in progress test ONLY branch. 
 c'mon I will not release the parent immediately...